packages/mobile/src/account/GoldEducation.tsx

```diff
@@ -1,63 +1,83 @@
+import { BtnTypes } from '@celo/react-components/components/Button.v2'
+import { navigate } from '@react-navigation/compat/lib/typescript/src/NavigationActions'
 import * as React from 'react'
-import { connect } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import { useDispatch, useSelector } from 'react-redux'
 import Education from 'src/account/Education'
 import CeloAnalytics from 'src/analytics/CeloAnalytics'
 import { CustomEventNames } from 'src/analytics/constants'
 import { setEducationCompleted } from 'src/goldToken/actions'
+import { Namespaces } from 'src/i18n'
 import { exchangeIcon, goldValue, shinyGold } from 'src/images/Images'
-import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { navigateBack } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
+import { RootState } from 'src/redux/reducers'
 
-interface DispatchProps {
-  setEducationCompleted: typeof setEducationCompleted
-}
-type Props = DispatchProps
-export class GoldEducation extends React.Component<Props> {
-  static navigationOptions = { header: null }
+export default function GoldEducation() {
+  const { t } = useTranslation(Namespaces.global)
+
+  const dispatch = useDispatch()
+
+  const isCeloEducationComplete = useSelector<RootState, boolean>(
+    (state) => state.goldToken.educationCompleted
+  )
 
-  goToExchange = () => {
-    this.props.setEducationCompleted()
+  const goToExchange = React.useCallback(() => {
```

Comment:
do we have any conventions around useCallback? i am not sure it's necessary here: https://kentcdodds.com/blog/usememo-and-usecallback